src/app/app-routing.module.ts

```diff
@@ -1,14 +1,19 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
 import { AppComponent } from './app.component';
+import { IndexComponent } from './components/index/index.component';
 import { ItinerarylistComponent } from './components/itinerary/itinerarylist/itinerarylist.component';
+import { NotFoundComponent } from './components/not-found/not-found.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7132249)
```suggestion
import { NotFoundComponent } from "./components/not-found/not-found.component";
```